data/tribes/initialization/atlanteans/units.lua

```diff
@@ -1473,45 +1513,80 @@ descriptions:new_tribe {
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Hunter's House
             purpose = pgettext("building", "Hunts animals to produce meat."),
             -- TRANSLATORS: Note helptext for an atlantean production site: Hunter's House
-            note = pgettext("atlanteans_building", "The hunter’s house needs animals to hunt within the work area.")
+            note = pgettext("atlanteans_building", "The hunter’s house needs animals to hunt within the work area."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site: Hunter's House
+            lore = pgettext("atlanteans_building", "Sometimes we end up in a region with no fish, so we must hunt. "..
+                                                   "Game will reproduce by itself, if not hunted too much. But yes, fish is our main meal."),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site: Hunter's House
+            lore_author = pgettext("atlanteans_building", "Hunter answering a fisher")
          }
       },
       {
          name = "atlanteans_well",
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Well
-            purpose = pgettext("building", "Draws water out of the deep.")
+            purpose = pgettext("building", "Draws water out of the deep."),
+            -- TRANSLATORS: Note helptext for an atlantean production site: Well
+            note = pgettext("atlanteans_building", "I hope you have checked this spot with a geologist."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site: Well
+            lore = pgettext("atlanteans_building", "Well, there is no wellbeing without a well."),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site: Well
+            lore_author = pgettext("atlanteans_building", "Water carrier muttering to himself")
          }
       },
       {
          name = "atlanteans_gold_spinning_mill",
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Gold Spinning Mill
-            purpose = pgettext("atlanteans_building", "Spins gold thread out of gold.")
+            purpose = pgettext("atlanteans_building", "Spins gold thread out of gold."),
+            -- TRANSLATORS: Note helptext for an atlantean production site: Gold Spinning Mill
+            note = pgettext("atlanteans_building", "Do not build until you have a steady gold supply."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site: Gold Spinning Mill
+            lore = pgettext("atlanteans_building", "No, this is not Rumpelstiltskin spinning gold from straw, we need real gold."),
```

Comment:
I agree with Wuzzy that using a character from German mythology for Atlanteans does not fit well.   
```suggestion
            lore = pgettext("atlanteans_building", "No, we can't spin goldthread from straw, we need real gold."),
```

Reply:
You really do not want to have _some_ fun?

Reply:
Well for me the Lore isn't about having fun. It is about explaining the tribes culture and background and by this creating more atmosphere in the game. So Atlantean Lore should reference only Atlantean mythology and traditions. 
We might add some familiar sayings, poetry and songs, but only in an atlantean version. So I implicitly tried to keep the theme by leaving the "spinning gold from straw" topic. But the explicit usage of a actual german tell-tale does not feel appropriate.

Reply:
You may have a look at the discussions about the barbarian Lore to see that it was the same there as well.      
Basic question is does it fit to the character and design of the tribe

Reply:
Hmmmm... How about adding: "Did you get that idea from some kind of Frisian fairy tale?" :))) if @Noordfrees doesn't object.